src/Backend/SgateShopgatePlugin/Models/Export/Product.php

```diff
@@ -656,6 +656,120 @@ public function getParentCategoryIds(\Shopware\Models\Category\Category $cat)
+            }
+        }
+
+        return false;
+    }
+
+    private function grabCustomizedAttributesForArticle($article)
+    {
+        $query = "
+            SELECT
+                `template`.`id` AS template_id,
+                `template`.`internal_name` AS template_internal_name,
+                `template`.`display_name` AS template_display_name,
+                `template`.`description` AS template_description,
+                `template`.`media_id` AS template_media_id,
+                `template`.`step_by_step_configurator` AS template_step_by_step_configurator,
+                `template`.`active` AS template_active,
+                `template`.`confirm_input` AS template_confirm_input,
+
+                `option`.`id` AS option_id,
+                `option`.`template_id` AS option_template_id,
+                `option`.`name` AS option_name,
+                `option`.`description` AS option_description,
+                `option`.`ordernumber` AS option_ordernumber,
+                `option`.`required` AS option_required,
+                `option`.`type` AS option_type,
+                `option`.`position` AS option_position,
+                `option`.`default_value` AS option_default_value,
+                `option`.`placeholder` AS option_placeholder,
+                `option`.`is_once_surcharge` AS option_is_once_surcharge,
+                `option`.`max_text_length` AS option_max_text_length,
+                `option`.`min_value` AS option_min_value,
+                `option`.`max_value` AS option_max_value,
+                `option`.`max_file_size` AS option_max_file_size,
+                `option`.`min_date` AS option_min_date,
+                `option`.`max_date` AS option_max_date,
+                `option`.`max_files` AS option_max_files,
+                `option`.`interval` AS option_interval,
+                `option`.`could_contain_values` AS option_could_contain_values,
+                `option`.`allows_multiple_selection` AS option_allows_multiple_selection,
+
+                `price`.`id` AS price_id,
+                `price`.`option_id` AS price_option_id,
+                `price`.`value_id` AS price_value_id,
+                `price`.`surcharge` AS price_surcharge,
+                `price`.`tax_id` AS price_tax_id,
+                `price`.`customer_group_name` AS price_customer_group_name,
+                `price`.`customer_group_id` AS price_customer_group_id,
+                `price`.`is_percentage_surcharge` AS price_is_percentage_surcharge,
+                `price`.`percentage` AS price_percentage
+
+            FROM
+                s_plugin_custom_products_template `template`
+                LEFT JOIN s_plugin_custom_products_template_product_relation `r` ON `r`.`template_id` = `template`.id
+                LEFT JOIN s_plugin_custom_products_option `option` ON `option`.`template_id` = `template`.id
+                LEFT JOIN s_plugin_custom_products_price `price` ON `price`.`option_id` = `option`.`id`
```

Comment:
These JOINs only capture simple input fields from the plugin (including file-uploads). Dropdown-based customized attributes are not supported.